### DISCUSSION TITLE
Open source file conversion

### DISCUSSION THREAD

**Discussion by mehrdadpirnazar** [artisan-roaster-scope/artisan#1577] [Q&A] [answered]
Hello good time I have downloaded the open source file folder and made some changes How can I convert it to an exe file?![IMG_20240608_005301.jpg](https://github.com/artisan-roaster-scope/artisan/assets/82454404/15e62684-0ca7-43fa-8be0-9ee309be9c2a)

**Comment by roasterdave** [comment, marked as answer]
https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/HowToBuildArtisan.md

**Reply by mehrdadpirnazar** [reply]
Hello good time
I am getting this error related to the one I sent the picture of
I have also set Github token in the settings.
![19_51_05_338114556-b2f7b73a-70c6-416a-92a4-423fbb9ba607.png](https://github.com/artisan-roaster-scope/artisan/assets/82454404/427e4dd5-9eaf-465e-ae96-1f08fc5c54fa)

**Reply by poundy** [reply]
yet again you come in and duplicate posts unnecessarily - didn't you learn etiquette from your previous thread https://github.com/artisan-roaster-scope/artisan/discussions/1549 ?

DO NOT DUPLICATE ASKING QUESTIONS. Anyone who can help will have seen your other post, and there was no need to duplicate it into this thread. Post it once, do not post it again, and do not think you've waited long enough and therefore ping people....

**Reply by roasterdave** [reply]
>$GITHUB_TOKEN missing, please set it in the Appveyor CI settings of this project

There is no valid token set in the Appveyor environment to allow uploading files to the Github repository.

Follow step 7 from the HowtoBuildArtisan document I linked above to set the token in Appveyor's settings.  
>7. To publish the build files to the GitHub releases page requires generating a GitHub personal access token and storing it in the AppVeyor project's environment. [Generate a new token](https://github.com/settings/tokens) with minimally repo:status, repo_deployment and public_repo permissions. Copy the token. Go now to [AppVeyor projects](https://ci.appveyor.com/projects), click on the project, then click Settings followed by Environment. Under "Environment Variables" click Add Variable. Set the name field to "GITHUB_TOKEN" and paste the token into the value field. Scroll to the bottom of the page and click Save.

**Reply by mehrdadpirnazar** [reply]
> >$GITHUB_TOKEN missing, please set it in the Appveyor CI settings of this project
> 
> There is no valid token set in the Appveyor environment to allow uploading files to the Github repository.
> 
> Follow step 7 from the HowtoBuildArtisan document I linked above to set the token in Appveyor's settings.  
> >7. To publish the build files to the GitHub releases page requires generating a GitHub personal access token and storing it in the AppVeyor project's environment. [Generate a new token](https://github.com/settings/tokens) with minimally repo:status, repo_deployment and public_repo permissions. Copy the token. Go now to [AppVeyor projects](https://ci.appveyor.com/projects), click on the project, then click Settings followed by Environment. Under "Environment Variables" click Add Variable. Set the name field to "GITHUB_TOKEN" and paste the token into the value field. Scroll to the bottom of the page and click Save.
> 

Hello, thank you for your reply
I have done this
But unfortunately I still get the same error

**Reply by roasterdave** [reply]
The message is telling you that the token is missing from your Appveyor project settings.  Please follow my step 7 closely.  Set the name field without the quotes.  There is nothing else I can suggest.
